crates/model/src/quote.rs

```diff
@@ -30,34 +30,66 @@ pub enum QuoteSigningScheme {
     Eip1271 {
         #[serde(rename = "onchainOrder")]
         onchain_order: bool,
+        #[serde(
+            rename = "verificationGasLimit",
+            default = "default_verification_gas_limit"
+        )]
+        verification_gas_limit: U256,
     },
     PreSign {
         #[serde(rename = "onchainOrder")]
         onchain_order: bool,
     },
 }
 
+impl QuoteSigningScheme {
+    pub fn new_eip1271_with_default_gas(onchain_order: bool) -> Self {
+        QuoteSigningScheme::Eip1271 {
+            onchain_order,
+            verification_gas_limit: default_verification_gas_limit(),
+        }
+    }
+}
+
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "camelCase")]
 struct QuoteSigningDeserializationData {
     #[serde(default)]
     signing_scheme: SigningScheme,
     #[serde(default)]
+    verification_gas_limit: Option<U256>,
```

Comment:
You probably need to specify decimal string serialization. Off the top of my head its something like:

```suggestion
#[serde_as]
#[derive(Debug, Deserialize)]
#[serde(rename_all = "camelCase")]
struct QuoteSigningDeserializationData {
    #[serde(default)]
    signing_scheme: SigningScheme,
    #[serde_as(as = "Option<DecimalU256>")]
    #[serde(default)]
    verification_gas_limit: Option<U256>,
```

Reply:
Fixed.